File: program/Primes/Primes.cs
using System;
using System.Linq;
using System.Collections.Generic;

class Primes {
	static void Main() {
		new List<int>{100}
		.SelectMany(n=>Enumerable.Range(1,(int)Math.Sqrt(n))
				.SelectMany(x=>Enumerable.Range(2,n/x-1)
					.Select(y=>x*y)))
			.GroupBy(x=>x)
			.Where(x=>x.Count()==1)
			.Select(x=>x.Key)
			.ToList()
			.ForEach(x=>Console.WriteLine(x));
	}
}

File: program/Primes/Primes.py
import math;
import itertools;

for x in [k for k, g in itertools.groupby(
          sorted([x*y for n in [100]
                  for x in range(1,int(math.sqrt(n))+1)
                  for y in range(2,n//x+1)]))
          if len(list(g))==1]:
    print(x)
